### PR TITLE
Add mixin for sidebar and menu

### PR DESCRIPTION
**Pull request by andreirtaylor**
Also fix warnings from mongo

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-var express = require('express');
-var router = express.Router();
+var express = require('express'),
+    router = express.Router(),
+    Choral = require('../models/chorals.js'),
```
Comment (cpjk) [inline routes/index.js:3]: forgot to check this in?

Comment (andreirtaylor) [reply routes/index.js:3]: Yup added it in

File: models/chorals.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+var mongoose = require('./setup');
+var Schema = mongoose.Schema;
+
+var choralSchema = new Schema({
+    choral_id: String,
+    children:[ String ],
```
Comment (cpjk) [inline models/chorals.js:6]: Should this be an object id? :thinking:

Comment (cpjk) [reply models/chorals.js:6]: err *array of*

**Comment by andreirtaylor** [comment]
Too many mistakes to merge.
I need to split this up better.

**Pull request closed without merging**